src/options/EncryptionOptions.ts

```diff
@@ -5,4 +5,5 @@ export interface EncryptionOptions {
   iv?: string;    //// For testing mainly.
   authTagLength?: number;
   looseMatching?: boolean;
+  encryptionPredicate?: (entity: any) => boolean;
```

Comment:
maybe change to `shouldEncrypt` and let it be a function or boolean value?

Reply:
I (respectfully) think we should not do that...

Because the term `predicate` implies a callable on a condition. `shouldEncrypt` with dual a usage (bool and a callback) as you propose would be less clear and complicates the if checks in the entity class. Lastly the whole intent is it should resolve to a boolean based on some condition. The condition should not be a hardcode false as that simply turn off encryption on column as a whole, and then why would a user use the encrypt at in the first place.

Unless you disagree with this of course I'm happy to make the change.